src/main/java/hu/dpc/phee/operator/entity/batch/Batch.java

```diff
@@ -16,6 +16,9 @@ public class Batch extends AbstractPersistableCustom<Long> {
     @Column(name = "BATCH_ID")
     private String batchId;
 
+    @Column(name = "SUB_BATCH_ID")
```

Comment:
Don't we have to provide a migration script for this?

Reply:
The migration script is inside operations-app, checkout below PR.

https://github.com/openMF/ph-ee-operations-app/pull/29